src/options.ts

```diff
@@ -153,9 +189,9 @@ const inferOptions = (inputs: Inputs): Options => {
     // use a different path than ~/.condarc to avoid messing up the user's condarc
     condarcFile: inputs.condarcFile
       ? path.resolve(untildify(inputs.condarcFile))
-      : path.join(path.dirname(PATHS.micromambaBin), '.condarc'), // next to the micromamba binary -> easier cleanup
+      : path.join(tempDirectory, '.condarc'), // next to the micromamba binary -> easier cleanup
     micromambaBinPath,
-    micromambaRunShellPath: path.join(path.dirname(micromambaBinPath), 'micromamba-shell'),
+    micromambaRunShellPath: path.join(tempDirectory, 'micromamba-shell'),
```

Comment:
Let's make this a subdirectory `path.join(tempDirectory, 'micromamba', 'micromamba-shell)`. Otherwise we will add all things in `${{ runner.temp }}` to `PATH`